Library/Homebrew/software_spec.rb

```diff
@@ -325,8 +326,8 @@ def select_download_strategy(specs)
 end
 
 class BottleSpecification
-  DEFAULT_PREFIX = "/usr/local".freeze
-  DEFAULT_CELLAR = "/usr/local/Cellar".freeze
+  DEFAULT_PREFIX = Homebrew::DEFAULT_PREFIX
```

Comment:
Maybe scrap this constant?

Reply:
`Linuxbrew/core` is currently using the idiom `HOMEBREW_PREFIX == BottleSpecification. DEFAULT_PREFIX` where Homebrew has been using `HOMEBREW_PREFIX == "/usr/local"`. So for compatibility this constant needs to remain for now. It could possibly be moved to `compat`.